packages/amplication-client/src/Entity/DeleteEntityField.tsx

```diff
@@ -30,8 +30,17 @@ export const DeleteEntityField = ({
   onDelete,
   onError,
 }: Props) => {
+  const { dataType, properties } = entityField;
+  const { relatedFieldId, relatedEntityId } = properties;
   const [confirmDelete, setConfirmDelete] = useState<boolean>(false);
   const { addEntity } = useContext(AppContext);
+  const { data } = useQuery(GET_ENTITY_WITH_SPECIFIC_FIELDS, {
```

Comment:
better use a more specific name
```suggestion
  const { data : relatedFieldData } = useQuery(GET_ENTITY_WITH_SPECIFIC_FIELDS, {
```